### PR TITLE
Added NortonTech handwired keypad (NortonTechPad)

### PR DESCRIPTION
**Pull request by NortonTech3D**
## Description

I have added a hardwired keyboard, the NortonTechPad, The changes made are for the config files for setting up via and a default key layout for the numpad.

There are more commits than I would like because this is my first project using git, but I have ensured that the only changes are for this new numpad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/nortontechpad/info.json
[the remark this answers is not in the document]
Comment (NortonTech-Official) [reply keyboards/handwired/nortontechpad/info.json:1]: Done

File: keyboards/handwired/nortontechpad/readme.md
[the remark this answers is not in the document]
Comment (NortonTech-Official) [reply keyboards/handwired/nortontechpad/readme.md:1]: Updated

**Comment by NortonTech-Official** [comment]
Suggestions have been implemented

File: keyboards/handwired/nortontechpad/info.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "keyboard_name": "NortonTechPad",
+    "manufacturer": "NortonTech",
+    "url": "",
+    "maintainer": "NortonTech",
+    "usb": {
+        "vid": "0xFEED",
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/info.json:7]: VIA will reject a keyboard using FEED as its vendorID

File: keyboards/handwired/nortontechpad/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_numpad_5x4(
+        KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/keymaps/default/keymap.c:23]:
```suggestion
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
```

File: keyboards/handwired/nortontechpad/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_numpad_5x4(
+        KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/keymaps/via/keymap.c:23]:
```suggestion
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
```

File: keyboards/handwired/nortontechpad/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# NortonTechPad
+
+![NortonTechPad]()
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/readme.md:3]: Add image or remove line

File: keyboards/handwired/nortontechpad/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+The NortonTechPad is a numpad inspired by the SiCK-Pad,it is also 3D printed and designed to be inexpensive.
+
+-   Keyboard Maintainer: [NortonTech](https://github.com/NortonTech-Official)
+-   Hardware Supported: Pro Micro
+-   Hardware Availability: [files to print and documentation]()
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/readme.md:9]: Add link or remove line

File: keyboards/handwired/nortontechpad/info.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+                {"matrix": [1, 0], "x": 0, "y": 1},
+                {"matrix": [1, 1], "x": 1, "y": 1},
+                {"matrix": [1, 2], "x": 2, "y": 1},
+                {"matrix": [1, 3], "x": 3, "y": 2, "h": 2},
+
+                {"matrix": [2, 0], "x": 0, "y": 2},
+                {"matrix": [2, 1], "x": 1, "y": 2},
+                {"matrix": [2, 2], "x": 2, "y": 2},
+
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/info.json:39]:
```suggestion

                {"matrix": [2, 0], "x": 0, "y": 2},
                {"matrix": [2, 1], "x": 1, "y": 2},
                {"matrix": [2, 2], "x": 2, "y": 2},
                {"matrix": [1, 3], "x": 3, "y": 2, "h": 2},

```

File: keyboards/handwired/nortontechpad/info.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D7", "E6", "B4", "B5"],
+        "rows": ["F7", "B1", "B3", "B2", "B6"]
+    },
+    "diode_direction": "COL2ROW",
+    "development_board": "promicro",
+    "layouts": {
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/info.json:23]:
```suggestion
    "community_layouts": ["numpad_5x4"],
    "layouts": {
```

File: keyboards/handwired/nortontechpad/info.json [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    "url": "",
+    "maintainer": "NortonTech",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x9801",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": false,
```
Comment (zvecr) [inline keyboards/handwired/nortontechpad/info.json:13]:
```suggestion
        "extrakey": true,
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/handwired/nortontechpad/keymaps/default/keymap.c (hunk 2 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/keymaps/default/keymap.c:23]: This keyboard does not use the community layout, the matrix/keymap will not work with this change

File: keyboards/handwired/nortontechpad/keymaps/via/keymap.c (hunk 3 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/keymaps/via/keymap.c:23]: This keyboard does not use the community layout, the matrix/keymap will not work with this change

File: keyboards/handwired/nortontechpad/info.json (hunk 6 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/info.json:39]: This keyboard does not use the community layout, the matrix/keymap will not work with this change

File: keyboards/handwired/nortontechpad/info.json (hunk 7 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/info.json:23]: This keyboard does not use the community layout, the matrix/keymap will not work with this change

File: keyboards/handwired/nortontechpad/info.json (hunk 1 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/info.json:7]: Do you have any suggestions how to get a better vendorID

File: keyboards/handwired/nortontechpad/info.json (hunk 7 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/info.json:23]: If I change the to the community layout it does not type correctly

File: keyboards/handwired/nortontechpad/info.json (hunk 6 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/info.json:39]: Thanks for the tip, I'll try this and see if works

For reference, the way I have the key layer out is the tall key is wired to col 3 and row 1, based on this do you think I should implement the changes Zvecr suggested?

Comment (zvecr) [reply keyboards/handwired/nortontechpad/info.json:39]: > I can't add on to zvecr's suggestion, but you should put the tall key on "y": 1

Updated suggestion to reflect this.

File: keyboards/handwired/nortontechpad/readme.md (hunk 4 above)
Comment (NortonTech3D) [reply keyboards/handwired/nortontechpad/readme.md:3]: done

**Review by drashna — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**